### PR TITLE
Bug 1395349 - use the production site for devel logins

### PR DESCRIPTION
**Pull request by djmitche**
Use the production site for development logins

### CODE REVIEW

File: src/auth/DevelopmentLoginMenuItem.js [hunk 1]
```diff
@@ -22,11 +22,16 @@ export default class DevelopmentLoginMenuItem extends React.PureComponent {
   onSelect() {
     const query = new URLSearchParams();
 
-    query.set('target', new URL('/login/development', window.location));
+    query.set('callback_url', new URL('/login/development', window.location));
+    query.set('scope', 'xyz'); // defeat the default '*' and require user to enter desired scopes
+    query.set('name', 'tools-devel');
```
Comment (helfi92) [inline src/auth/DevelopmentLoginMenuItem.js]: Should we set the expiry date in the query or simply default to 3 days?

File: src/auth/DevelopmentLoginMenuItem.js [hunk 2]
```diff
@@ -22,11 +22,16 @@ export default class DevelopmentLoginMenuItem extends React.PureComponent {
...
     query.set(
       'description',
-      'Taskcluster Tools offers various way to create and inspect both tasks and task groups.'
+      'Temporary client for use developing and testing taskcluster-tools'
```
Comment (helfi92) [inline src/auth/DevelopmentLoginMenuItem.js]: s/use//

File: src/views/DevelopmentLogin/index.js [hunk 3]
```diff
@@ -11,8 +11,7 @@ export default class DevelopmentLogin extends React.Component {
     this.state = {
       credentials: {
         clientId: search.get('clientId'),
-        accessToken: search.get('accessToken'),
-        certificate: search.get('certificate')
```
Comment (helfi92) [inline src/views/DevelopmentLogin/index.js]: Out of curiosity, why do we not need certificate anymore?

File: src/auth/DevelopmentLoginMenuItem.js (hunk 1 above)
Comment (djmitche) [reply src/auth/DevelopmentLoginMenuItem.js]: I'm OK with the default.

File: src/views/DevelopmentLogin/index.js (hunk 3 above)
Comment (djmitche) [reply src/views/DevelopmentLogin/index.js]: These are now "permacreds" and not temporary credentials.

Comment (helfi92) [reply src/views/DevelopmentLogin/index.js]: Sounds good, thanks!

**Comment by eliperelman** [comment]
This is only during development, so no risk. Merge when convenient.

**Pull request merged**